tests/UI/pages/BO/advancedParameters/email/index.js

```diff
@@ -200,12 +212,19 @@ class Email extends BOBasePage {
       this.waitForVisibleSelector(page, `${this.confirmDeleteModal}.show`),
     ]);
     await this.clickAndWaitForNavigation(page, this.confirmDeleteButton);
+
     return this.getAlertSuccessBlockParagraphContent(page);
   }
 
-  /** setup the smtp parameters
-   *
-   * @returns {Promise<void>}
+  /**
+   * Setup the smtp parameters
+   * @param page {Page} Browser tab
+   * @param server {string} Value of server name to set on smtp server input
+   * @param username {string} Value of username to set on smtp username input
+   * @param pass {string} Value of password to set on smtp password input
+   * @param port {string} Value of port to set on smtp port input
+   * @param encryption {string} Value of encryption to select from smtp encryption select
```

Comment:
it's a lot of parameters, I think we should gather them on 1 parameter called `smtpConfig`.
But It's another subject for another PR, we have just to keep it in mind.

Reply:
I approve and use decompositions like `{page, server, username, pass, port, encryption}` will permit to track BC easier (in case tests are opened to the public)